src/_style.scss

```diff
@@ -564,3 +564,20 @@ select.form-control {
   color: $gray-700;
   text-decoration: none;
 }
+
+.dropdown-item:active {
+    background-color: #F2F0EF;
+}
+
+.dropdown-container {
+    box-shadow: 3px 3px 15px #F2F0EF;
+    max-height: 250px;
+    overflow-y: scroll;
+    position: absolute;
+    background-color: #fff;
+    width: 464px;
+}
+
+.d-none {
+display:none;
+}
```

Comment:
Remove this if not in use otherwise you can use already available `hidden` class.